Collections/Player Utilities/conc.alias

```diff
@@ -28,6 +28,10 @@ if c and (target := c.get_combatant(target)):
  {concRoll}; {["Failure! Concentration Dropped", "Success!"][success]}" """
  if not success:
   if effect:
+   concOut   = concOut[:-2] + f"""\nYou are no longer concentrating on {effect.name} and the effects have been removed." """
```

Comment:
This line is duplicated on line 36 now, so It'll show twice?

Reply:
That line was on the original alias, so I didn't touch it